application-xpoll-ui/src/main/resources/XPoll/XPollSheet.xml

```
@@ -172,7 +172,7 @@
 #end
 ## Flag to know wheter the options must be displayed as checkbox or radio inputs.
 #set ($isRadio = $pollObj.getProperty('type').value == 'single')
-#if ($xcontext.action == 'edit' || $status == 'inpreparation')
+#if (($xcontext.action == 'edit' || $status == 'inpreparation') &amp;&amp; $hasEdit)
```

Maybe check $hasEdit first to fail faster